app/Models/Location/City.php

```diff
@@ -56,27 +56,29 @@ class City extends Location
      */
     public function getTotalLitterRedisAttribute ()
     {
-        return Redis::hexists("city:$this->id", "total_litter")
+        return (Redis::hexists("city:$this->id", "total_litter"))
             ? (int)Redis::hget("city:$this->id", "total_litter")
-            : 0;
+            : 1;
```

Comment:
We should not modify this, if there's no litter for that key, then simply return 0. To avoid that nasty division by zero error we should make checks where we use this property, for example, MapController, line 68.